src/openqaoa-core/qaoa_components/ansatz_constructor/baseparams.py

```diff
@@ -5,6 +5,7 @@
 
 from .operators import Hamiltonian
 from .hamiltonianmapper import HamiltonianMapper
+from .gatemap import RotationGateMap
```

Comment:
@vishal-ph this should fix the doc issue ... up to you if we want to find an alternative solution